src/PrestaShopBundle/Form/Admin/Sell/Supplier/SupplierType.php

```diff
@@ -103,17 +118,36 @@ public function __construct(
         $this->statesChoiceProvider = $statesChoiceProvider;
         $this->contextCountryId = $contextCountryId;
         $this->isMultistoreEnabled = $isMultistoreEnabled;
+        $this->router = $router;
     }
 
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $data = $builder->getData();
         $countryId = 0 !== $data['id_country'] ? $data['id_country'] : $this->contextCountryId;
-        $stateChoices = $this->statesChoiceProvider->getChoices(['id_country' => $countryId]);
+
+        $invalidCharsText = sprintf(
+            '%s ' . TypedRegexValidator::GENERIC_NAME_CHARS,
+            $this->trans('Invalid characters:', 'Admin.Notifications.Info')
```

Comment:
```suggestion
            $this->trans('Invalid characters:', 'Admin.Global')
```

Reply:
Sorry I missed this one but I corrected it in other PRs so I'll do it here too for more consistency. 👍